[2025-06-02T18:27:59.657+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_docker_dag.run_test_task scheduled__2023-05-05T00:00:00+00:00 [queued]>
[2025-06-02T18:27:59.702+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_docker_dag.run_test_task scheduled__2023-05-05T00:00:00+00:00 [queued]>
[2025-06-02T18:27:59.709+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-06-02T18:27:59.769+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_test_task> on 2023-05-05 00:00:00+00:00
[2025-06-02T18:27:59.858+0000] {standard_task_runner.py:60} INFO - Started process 569 to run task
[2025-06-02T18:27:59.887+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'test_docker_dag', 'run_test_task', 'scheduled__2023-05-05T00:00:00+00:00', '--job-id', '140', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmphuea877j']
[2025-06-02T18:27:59.938+0000] {standard_task_runner.py:88} INFO - Job 140: Subtask run_test_task
[2025-06-02T18:28:00.163+0000] {task_command.py:423} INFO - Running <TaskInstance: test_docker_dag.run_test_task scheduled__2023-05-05T00:00:00+00:00 [running]> on host c22acf1bbdfe
[2025-06-02T18:28:00.676+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_docker_dag' AIRFLOW_CTX_TASK_ID='run_test_task' AIRFLOW_CTX_EXECUTION_DATE='2023-05-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-05T00:00:00+00:00'
[2025-06-02T18:28:00.698+0000] {logging_mixin.py:188} INFO - âœ… Airflow DAG ran successfully inside Docker! After updating
[2025-06-02T18:28:00.714+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-02T18:28:00.756+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=test_docker_dag, task_id=run_test_task, execution_date=20230505T000000, start_date=20250602T182759, end_date=20250602T182800
[2025-06-02T18:28:00.843+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-02T18:28:00.945+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
